PEN-610: Metadata for search page

beltrancaliz opened the pull request:
Fixed metadata for search page
[Jira ticket](https://arcpublishing.atlassian.net/browse/PEN-610)

beltrancaliz committed:
Metadata for search page

JackHowa reviewed (approved):
If you have extra time in this sprint, this logic is a prime case for unit testing

beltrancaliz committed:
Merge branch 'master' into PEN-610-Search-Metadata

beltrancaliz committed:
Publish

 - @wpmedia/article-body-block@2.0.6
 - @wpmedia/article-tag-block@2.0.6
 - @wpmedia/author-bio-block@1.0.6
 - @wpmedia/author-content-source-block@1.0.6
 - @wpmedia/byline-block@1.0.6
 - @wpmedia/card-list-block@1.0.6
 - @wpmedia/collections-content-source-block@1.0.6
 - @wpmedia/content-api-source-block@1.0.6
 - @wpmedia/date-block@2.0.6
 - @wpmedia/default-output-block@2.1.0
 - @wpmedia/double-chain-block@1.0.6
 - @wpmedia/extra-large-manual-promo-block@1.0.6
 - @wpmedia/extra-large-promo-block@1.0.6
 - @wpmedia/footer-block@2.0.6
 - @wpmedia/full-author-bio-block@1.0.6
 - @wpmedia/gallery-block@1.0.6
 - @wpmedia/header-nav-block@2.0.6
 - @wpmedia/headline-block@2.0.6
 - @wpmedia/htmlbox-block@1.0.6
 - @wpmedia/large-manual-promo-block@1.0.6
 - @wpmedia/large-promo-block@1.0.6
 - @wpmedia/lead-art-block@2.0.6
 - @wpmedia/links-bar-block@2.0.6
 - @wpmedia/masthead-block@1.0.6
 - @wpmedia/medium-manual-promo-block@1.0.6
 - @wpmedia/medium-promo-block@1.0.6
 - @wpmedia/numbered-list-block@3.0.6
 - @wpmedia/overline-block@1.0.6
 - @wpmedia/results-list-block@2.0.6
 - @wpmedia/right-rail-block@1.0.6
 - @wpmedia/search-content-source-block@1.0.6
 - @wpmedia/search-results-list-block@0.2.1
 - @wpmedia/section-title-block@1.1.1
 - @wpmedia/share-bar-block@2.0.6
 - @wpmedia/shared-styles@1.0.6
 - @wpmedia/simple-list-block@2.0.6
 - @wpmedia/single-chain-block@1.0.6
 - @wpmedia/site-hierarchy-content-block@3.1.5
 - @wpmedia/small-manual-promo-block@1.0.6
 - @wpmedia/small-promo-block@1.0.6
 - @wpmedia/story-feed-author-content-source-block@1.0.6
 - @wpmedia/story-feed-query-content-source-block@1.0.6
 - @wpmedia/story-feed-sections-content-source-block@1.0.6
 - @wpmedia/story-feed-tag-content-source-block@1.0.6
 - @wpmedia/subheadline-block@2.0.6
 - @wpmedia/tag-title-block@1.0.6
 - @wpmedia/tags-content-source-block@1.0.6
 - @wpmedia/top-table-list-block@1.0.6
 - @wpmedia/triple-chain-block@1.0.6
 - @wpmedia/unpublished-content-source-block@1.0.6
 - @wpmedia/video-player-block@1.0.6
